Source/ZoomNet/Models/WebhookAiCompanionMeetingTranscript.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Represents meeting and transcript file information as received in <see cref="Webhooks.MeetingAicTranscriptCompletedEvent"/>.
+	/// </summary>
+	public class WebhookAiCompanionMeetingTranscript
```

Comment:
Is this class intended to be used exclusively for webhook content parsing? If so, it should be under the `ZoomNet.Models.Webhooks` namespace.

However, it's perfectly fine to leave this class in the `ZoomNet.Models` namespace if you think we can use this class to parse data returned from the API.